### ISSUE TITLE
Composer install error and setting up in remotely accessible localhost server issue

### ISSUE THREAD

**Issue by moniruss** [snipe/snipe-it#1575]
Hi, every time I try installing composer, its giving downloading error in midway! Tried it in several machines. I am giving a snapshot here:
![error1](https://cloud.githubusercontent.com/assets/16552454/12109158/a269fece-b3a9-11e5-88ef-44f6957c4935.png)

Another issue is i have set up a computer with an ip(say 192.168.255.29) and there i am using xampp (besides IIS) to run server with port 10 (say: 192.168.255.29:10 thats what i put in my laptop to access it remotely through similar network). now what will be recommended environment? local or production? and what will be the changes in configuration in bootstrap/start.php, app/config/(local or production folder?)/(app.php, database.php)
There's also this thing with the key. How to generate it? from command prompt? but there it says something like no artisan found and etc. So let me know please asap and guide me to somewhere if there's already this issue specifically fixed.

**Comment by snipe** [comment]
http://docs.snipeitapp.com/common-issues.html#during-composer-install-its-asking-me-for-my-github-credentials

Either do what it says and generate a Github token, or use `--prefer-source`.

Run it in production. You won't need to change `bootstrap/start.php`, and use the `app/config/production` files. Per the docs:

http://docs.snipeitapp.com/installation/configuration.html

```
 Unless you are a PHP developer setting up a local installation so that you can 
contribute code towards Snipe-IT, you should be running Snipe-IT in production mode.
```

The key gets generated when you run the installer, or you can generate it manually, per the docs. http://docs.snipeitapp.com/installation/configuration.html#application-settings

**Comment by moniruss** [comment]
Hi again.
I have done everything and it seemed it installed, asked for user, password and everything got loaded. But I can't access it. I renamed the folder to "asset" so its like"localhost/asset" to access. I tried following sub-directory instruction in the documentation but not sure why its not working. When I try to access via browser, it shows all the containing files and folders instead. How to run the sute? (Excuse me if its silly asking' i ain't very familiar with this)
